src/ducks/pin/PinEditView.jsx

```diff
@@ -14,6 +16,35 @@ import { pinSetting } from 'ducks/pin/queries'
 import { SETTINGS_DOCTYPE } from 'doctypes'
 import styles from 'ducks/pin/styles.styl'
 import { PIN_MAX_LENGTH } from 'ducks/pin/constants'
+import fingerprint from 'assets/icons/icon-fingerprint.svg'
+
+const FingerprintChoice = translate()(({ t, onChoice }) => {
+  return (
+    <div className={styles.Pin__FingerprintChoice}>
+      <div className={styles.Pin__FingerprintChoice__top}>
+        <Icon size="4rem" icon={fingerprint} />
+        <br />
+        {t('Pin.use-fingerprint-question')}
+        <br />
+      </div>
+      <div style={{ flexGrow: 0 }}>
```

Comment:
Is it possible to avoid inline styles ? I'm not totally against it, but I'd rather not mix classnames and inline styles

Reply:
Done via utility class in https://github.com/cozy/cozy-banks/pull/1103/commits/41374e2a30ac7ccb74bcb54aa2f406f955102168 . Thanks !